### ISSUE TITLE
Keyboard navigation and editing in code area does not work on the last 2 or 3 lines of text.

### ISSUE THREAD

**Issue by thinsoldier** [FreeCodeCamp/FreeCodeCamp#9171]
This has happened in the last 5 html exercises.

These are lines 42 to 46.
<ol>
  <li>flea treatment</li>
  <li>thunder</li>
  <li>other cats</li>
</ol>

If I click within the last LI's text my insertion point appears within the LI tag itself. If I then press a letter key the letter will appear after the closing OL tag. After pressing backspace to delete the error the code editor behaves normally.

**Comment by raisedadead** [comment]
This is a possible duplicate of #7847. To be re-opened if otherwise.
